src/deny-list/ip-deny-list.ts

```diff
@@ -83,6 +83,7 @@ export const updateIpDenyList = async (
 
   // delete the old ip deny list and create new one
   transaction.del(ipDenyList)
+  // @ts-expect-error ...allIps type works but gets type error
```

Comment:
Are you sure it gives a type error? When I checked I saw `Unused '@ts-expect-error' directive.`. Which seems to be correct comparing method type and `allIps` types